apps/avatax/graphql/fragments/User.graphql

```diff
@@ -1,5 +1,5 @@
 fragment User on User {
   id
   email
-  avataxCustomerCode: metafield(key: "avataxCustomerCode")
+  legacyAvataxCustomerCode: metafield(key: "legacyAvataxCustomerCode")
```

Comment:
not sure if we should change the name. we can make it `@deprecated` in the code. 
eg if we have failed async event delivery, we can have problem with old aliast being delivered

Reply:
Good idea - I changed it back 😄